inspirehep/modules/orcid/utils.py

```diff
@@ -209,3 +216,26 @@ def canonicalize_xml_element(element):
         exclusive=True,
     )
     return output_stream.getvalue()
+
+
+def get_literature_recids_for_orcid(orcid):
+    orcid_object = '[{"schema": "ORCID", "value": "%s"}]' % orcid
+
+    try:
+        author_recid = db.session.query(PersistentIdentifier.pid_value)\
+                         .filter(PersistentIdentifier.object_uuid == RecordMetadata.id)\
+                         .filter(PersistentIdentifier.pid_type == 'aut')\
+                         .filter(type_coerce(RecordMetadata.json, JSONB)['ids'].contains(orcid_object))\
+                         .one().pid_value
+    except MultipleResultFound as e:
+        LOGGER.error('Multiple author records found for ORCiD {}'.format(orcid))
+        six.reraise(e)
+    except NoResultFound as e:
+        LOGGER.error('No author record found for ORCiD {}'.format(orcid))
+        six.reraise(e)
```

Comment:
@puntonim this, or just remove the error handling from this function and let it blow?

Reply:
Remove entirely the error handling.
This code writes twice to Sentry (one with the LOGGER and one with reraise). What is the advantage of this? If it's the error message then I guess you can do six.raise_from or edit the message in e.